### PR TITLE
sync README.md files with the current state of the code

### LINKED ISSUE #356: Review README files
Some files got outdated. For example `cmd/gpu_plugin/README.md` describes the installation procedure like we still use `dep` instead of `go mod`.

### LINKED ISSUE #447: Documentation does not mention operator support for FPGA
Current documentation is not describing FPGa operator support e.g. here:

Currently the operator has limited support for the QAT and GPU device plugins: it validates container image references and extends reported statuses.

We should check documentation is updated in all the relevant places.

### PR DESCRIPTION
**Pull request by rojkov**
Closes #356
Closes #447

### CODE REVIEW

**Review by bart0sh — APPROVED** [review]

File: README.md
Comment (mythi) [inline README.md:122]: Closes #447  too

**Review by mythi — APPROVED** [review]

**Comment by mythi** [comment]
let's wait for @uniemimu's approval too ;)

**Review by uniemimu — APPROVED** [review]

**Pull request merged**
